blueprints/empress-blog/files/__base__/tag/getting-started.md

```diff
@@ -2,6 +2,8 @@
 name: Getting Started
 image:
 imageMeta:
+meta:
```

Comment:
same as above: move this to the dummy app

Reply:
Same answer as above.